app/controllers/orders_controller.rb

```diff
@@ -35,6 +35,13 @@ def index
     @user = current_user
   end
 
+  def update
+    order = current_user.orders.find(params[:id])
+    order.cancel_process
+    redirect_to '/profile/orders'
+    flash[:notice] = 'Order has been cancelled.'
+  end
```

Comment:
Cancel_process is in order model: allows all item_orders to have their status switched to unfulfilled and changes order status to cancelled.